### PR TITLE
gwe: rename to greenwithenvy, 0.15.4 -> 0.15.5

### PR DESCRIPTION
**Pull request by IvarWithoutBones**
###### Description of changes
Renamed it to match upstream a bit better, and added a few small improvements. See [the changelog](https://gitlab.com/leinardi/gwe/-/releases/0.15.5) for more information.

Supercedes: #167230

###### Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and “core” functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by WolfangAukang** [comment]
Result of `nixpkgs-review pr 167562` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li>greenwithenvy</li>
  </ul>
</details>

File: pkgs/tools/misc/greenwithenvy/default.nix [hunk 1]
```diff
@@ -1,6 +1,8 @@
 { lib
 , stdenv
 , fetchFromGitLab
+, testVersion
```
Comment (davidak) [inline pkgs/tools/misc/greenwithenvy/default.nix:4]:
```suggestion
, testers
```

this was renamed https://github.com/NixOS/nixpkgs/commit/f1c7f19e49190cf58041908f9c194431e2513688

File: pkgs/tools/misc/greenwithenvy/default.nix [hunk 2]
```diff
@@ -68,20 +70,26 @@ in stdenv.mkDerivation rec {
...
 
     makeWrapper ${pythonEnv}/bin/python $out/bin/gwe \
       --add-flags "$out/lib/gwe-bin" \
       --prefix LD_LIBRARY_PATH : "/run/opengl-driver/lib" \
       --prefix PATH : "${builtins.concatStringsSep ":" [ (lib.makeBinPath [ nvidia_x11 nvidia_x11.settings ]) "/run/wrappers/bin" ]}" \
-      --unset "SHELL" \
       ''${gappsWrapperArgs[@]}
   '';
 
+  passthru.tests.version = testVersion { package = greenwithenvy; };
```
Comment (davidak) [inline pkgs/tools/misc/greenwithenvy/default.nix:86]:
```suggestion
  passthru.tests.version = testers.testVersion { package = greenwithenvy; };
```

**Comment by IvarWithoutBones** [comment]
Marking this as a draft for now as I both have to rebase, and after checking repology it turns out all other package repositories do just call it `gwe`. I'll fix up this PR soon 👍

**Pull request closed without merging**